modules/web/js/ballerina/components/transform-render.js

```diff
@@ -576,13 +582,14 @@ class TransformRender {
 
         if (type == 'source') {
             subPlaceHolder = 'leftType';
-            $(`#${this.placeHolderName}`).find('.leftType').append(newVar);
-            this.addSource(property, this, true);
+            $('#' + this.placeHolderName).find('.leftType').append(newVar);
+            this.addSource(property, this, false);
```

Comment:
@hemikak : true -> false is purposely done and same line has been refactored by linter too.